docs/index.md

```diff
@@ -101,12 +101,17 @@ As [measured in April 2020](https://github.com/kreier/solarmeter/blob/master/REA
 
 Further measurement has been conducted in November 2020. The results.
 
+- Running 66 mA
+- WiFi 80-140 mA
+- CPU 80 MHz 48 mA
+- Light Sleep 9 mA
```

Comment:
There might be some issues with deep sleep. The power consumption seems not to drop.